[controller module] Could not open [null\conf\logback_controller.xml]

1. Please describe the issue you observed:
when I start Controller module, and not config rocketmqHome . then throw Exception
```
ch.qos.logback.core.joran.spi.JoranException: Could not open [null\conf\logback_controller.xml].
	at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:85)
	at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:72)
	at org.apache.rocketmq.controller.ControllerStartup.createControllerManager(ControllerStartup.java:112)
	at org.apache.rocketmq.controller.ControllerStartup.main0(ControllerStartup.java:57)
	at org.apache.rocketmq.controller.ControllerStartup.main(ControllerStartup.java:51)
Caused by: java.io.FileNotFoundException: null\conf\logback_controller.xml (系统找不到指定的路径。)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:80)
	... 4 more
```
hope to pre-check,if not set rocketmqHome  should Give a prompt and exit the system
![image](https://user-images.githubusercontent.com/15797831/181207089-4c288d72-48dd-478d-acb7-2cc17e79f9c6.png)

--- a/controller/src/main/java/org/apache/rocketmq/controller/ControllerStartup.java
+++ b/controller/src/main/java/org/apache/rocketmq/controller/ControllerStartup.java
@@ -29,6 +29,7 @@ import org.apache.commons.cli.CommandLine;
 import org.apache.commons.cli.Option;
 import org.apache.commons.cli.Options;
 import org.apache.commons.cli.PosixParser;
+import org.apache.commons.lang3.StringUtils;
 import org.apache.rocketmq.common.MixAll;
 import org.apache.rocketmq.common.constant.LoggerName;
 import org.apache.rocketmq.common.ControllerConfig;
@@ -109,6 +110,11 @@ public class ControllerStartup {
         JoranConfigurator configurator = new JoranConfigurator();
         configurator.setContext(lc);
         lc.reset();
+
+        if (StringUtils.isEmpty(controllerConfig.getRocketmqHome())) {
+            System.out.printf("Please set the %s or %s variable in your environment!%n", MixAll.ROCKETMQ_HOME_ENV, MixAll.ROCKETMQ_HOME_PROPERTY);
+            System.exit(-1);
+        }
         configurator.doConfigure(controllerConfig.getRocketmqHome() + "/conf/logback_controller.xml");
 
         log = InternalLoggerFactory.getLogger(LoggerName.NAMESRV_LOGGER_NAME);

